CakePHP/Sniffs/NamingConventions/ValidClassBracketsSniff.php

```diff
@@ -19,27 +19,32 @@
  * Ensures curly brackets are on the same line as the Class declaration
```

Comment:
Right, it is in fact handle "more" nicely with the PSR2 and the autofixer. I've removed it.